librb/include/rb_memory.h

```diff
@@ -80,26 +80,29 @@ rb_strdup(const char *x)
 	return (ret);
 }
 
-
+// overrule libc's prototype and ignore the cast warning, allowing proper const
```

Comment:
This allows destruction of pointers with pointed-to data that should otherwise be const (and const char* is a ubiquitous case) see: http://yarchive.net/comp/const.html